modules/core/src/main/java/org/apache/fluo/core/thrift/OracleService.java

```diff
@@ -26,6 +26,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@SuppressWarnings({"rawtypes", "unused", "serial", "unchecked"})
```

Comment:
The change is not fixing a bug or changing behavior, so regenerating and losing this change should not introduce any bugs.  Based on this I think its ok.